### ISSUE TITLE
Asked IRL: You are not an SRE, why do you invest your time in supporting SLXs and Error Budgets?

### ISSUE THREAD

**Issue by estherthetester** [estherthetester/ama#3]

**Comment by estherthetester** [comment]
**TL;DR: SLxs are a great way to measure the quality of a service and to have informed discussions on where to invest. They also support testing in production.**

While I am yet to experience working in an organisation that has fully adopted the [SRE](https://medium.com/@rakyll/the-sre-model-6e19376ef986) model, I have experimented with adopting SRE concepts previously and find they are complementary to data driven testing and testing in production.
 
There is a whole post I can write on this subject. But essentially it boils down to:

"If you cannot measure it, then you cannot improve it.” _- Lord Kelvin, Sir William Thomson_
"Hope is not a strategy.” _- Mentioned in the latest Mission Impossible_

More reading:
* https://landing.google.com/sre/book.html
* https://www.amazon.co.uk/Site-Reliability-Workbook-Practical-Implement/dp/1492029505
